worldedit-core/src/main/java/com/sk89q/worldedit/EditSession.java

```diff
@@ -242,20 +248,19 @@ public String getDisplayName() {
     //FAWE start
     private final Relighter relighter;
     private final boolean wnaMode;
-
     @Nullable
     private final Region[] allowedRegions;
+    private @Nullable List<TracingExtent> tracingExtents = null;
```

Comment:
Put this above the Fawe annotation or close it. Also, any reason why this shouldn't be final? The initializer isn't needed.